Modix.Services/UserInfo/UserInfoEmbedBuilderHelper.cs

```diff
@@ -0,0 +1,167 @@
+            return this;
+        }
+
+        public UserInfoEmbedBuilderHelper WithMemberInformation(IGuildUser user)
+        {
+            _content.AppendLine("\n**\u276F Member Information**");
+
+            if (!string.IsNullOrEmpty(user.Nickname))
+            {
+                _content.AppendLine("Nickname: " + user.Nickname);
+            }
+
+            _content.AppendLine($"Created: {FormatUtilities.FormatTimeAgo(_nowUtc, user.CreatedAt)}");
+
+            if (user.JoinedAt != null)
+            {
+                _content.AppendLine($"Joined: {FormatUtilities.FormatTimeAgo(_nowUtc, user.JoinedAt.Value)}");
+            }
+
+            if (user.RoleIds?.Count > 0)
+            {
+                var roles = user.RoleIds.Select(x => user.Guild.Roles.Single(y => y.Id == x))
+                    .Where(x => x.Id != x.Guild.Id)
+                    .OrderByDescending(role => role)
+                    .ToArray();
+
+                if (roles.Any())
+                {
+                    _content.Append($"{"Role".ToQuantity(roles.Length, ShowQuantityAs.None)}: ");
+                    _content.AppendLine(roles.Select(r => r.Mention).Humanize());
+                }
+            }
+
+            return this;
+        }
+
+        public UserInfoEmbedBuilderHelper WithPromotions(List<(ulong targetRoleId, DateTimeOffset date)> promotions)
+        {
+            _content.AppendLine("\n**\u276F Promotion History**");
+
+            foreach (var promotion in promotions.OrderByDescending(x => x.date))
+            {
+                _content.AppendLine($"• {MentionUtils.MentionRole(promotion.targetRoleId)} {FormatUtilities.FormatTimeAgo(_nowUtc, promotion.date)}");
+            }
+
+            return this;
+        }
+
+        public UserInfoEmbedBuilderHelper WithInfractions(string websiteBaseUrl, ulong userId,
+            IGuildChannel channel, Dictionary<InfractionType, int> infractions)
+        {
+            // https://modix.gg/infractions?subject=1234
+            var url = new UriBuilder(websiteBaseUrl)
+            {
+                Path = "/infractions",
+                Query = $"subject={userId}"
+            }.RemoveDefaultPort().ToString();
+
+            _content.AppendLine();
+            _content.AppendLine($"**\u276F Infractions [See here]({url})**");
+
+            if (!channel.IsPublic())
+            {
+                _content.AppendLine(FormatUtilities.FormatInfractionCounts(infractions));
+            }
+            else
+            {
+                _content.AppendLine("Infractions cannot be listed in public channels.");
+            }
+
+            return this;
+        }
+
+        public string Build()
+        {
+            return _content.ToString();
+        }
+
+        private string GetParticipationEmoji(int rank) =>
+            rank switch
+            {
+                1 => "🥇",
+                2 => "🥈",
+                3 => "🥉",
+                _ => "🏆",
```

Comment:
We currently only show a trophy if the user is in the top 90th percentile of active users or higher, but this would show the trophy for everybody. It looks like you have the percentile [in the method where this is being called](https://github.com/discord-csharp/MODiX/pull/678/files#diff-deea26e023c5e436db56444765a0d7dbR71) - so was this an intentional change?